java/code/src/com/suse/manager/webui/services/SaltStateGeneratorService.java

```diff
@@ -193,6 +193,58 @@ private String getImagePillarFileName(OSImageInspectSlsResult.Bundle bundle) {
                 bundle.getId().replace('.', '-') + "." + PILLAR_IMAGE_DATA_FILE_EXT;
     }
 
+    public void createImageSyncedPillar(ServerGroup branch, String name, String version) {
+
+        try {
+            SaltPillar pillar = new SaltPillar();
+            Map<String, Object> imagePillar = new TreeMap<String, Object>();
```

Comment:
Any particular reason to use TreeMap instead of HashMap? Do you need to maintain order? Moreover, if a map is going to have only one entry, may be better user collections's singletonMap method like [here](https://github.com/SUSE/spacewalk/blob/ff22e6e1ca1c1ab5548510b80243d73b9920ddf4/java/code/src/com/redhat/rhn/manager/formula/test/FormulaManagerTest.java#L227)

Reply:
fixed